core/src/components/progress-bar/test/shape/progress-bar.e2e.ts

```diff
@@ -0,0 +1,56 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+configs({ modes: ['ionic-md'] }).forEach(({ title, screenshot, config }) => {
+  test.describe(title('progress-bar: shape'), () => {
+    test('should render a round progress bar', async ({ page }) => {
```

Comment:
Let's keep it consistent with other shape tests:

```suggestion
    test('round - should not have visual regressions', async ({ page }) => {
```